app/controllers/organisation/mission_controller.rb

```diff
@@ -33,19 +33,18 @@ def update
   end
 
   private
+
   def organisation_params
 
     # When no checkbox is selected on the page no mission key/value is passed
     # in the form, meaning that the organisation hash is no longer passed
     # through either. In this case, we need to add it manually to avoid
     # triggering a 'param is missing or value is empty' exception
-    unless params[:organisation].present?
-      params.merge!({organisation: {mission: ""}})
-    end
+
+    params.merge!({ organisation: { mission: '' } }) if params[:organisation].blank?
```

Comment:
```suggestion
    params.merge!({ organisation: { mission: '' } }) unless params[:organisation].present?
```

I'm not convinced by the change in logic here - we've kept to `.present?` elsewhere when addressing RuboCop violations.

I'll test locally and then commit the suggestion if it doesn't re-trigger a violation.